### DISCUSSION TITLE
关于帖子数量

### DISCUSSION THREAD

**Discussion by Bai6869** [share121/inter-knot#2561] [其他]
虽然绳网很有意思 但是感觉帖子的更新速度和数量并不多啊（主要是已经刷的差不多想看新的<img src="https://img-static.mihoyo.com/communityweb/upload/a7304904a31329464114b1074218967e.png" alt="帕姆 瘫倒" width="75" >

**Comment by HifumiMizuhara** [comment]
帖子数量要咱们自己去创造！

**Reply by Bai6869** [reply]
居然是这样解决的吗？大师，我悟了！

**Comment by Natsumowari** [comment]
这个委托我接了！
